puppet/modules/slave/manifests/init.pp

```diff
@@ -143,6 +143,8 @@
       ensure => present;
     'ansible':
       ensure => latest;
+    'virtualenv':
```

Comment:
Did you check that this package exists and you can install it by this name on centos box?

Reply:
Both Debian and EPEL call it `python-virtualenv`